SharedSource/UnitTests/Utilities/AssertionTest.cs

```diff
@@ -135,7 +136,7 @@ public void IsNotNull_True ()
     public void IsNotNull_False ()
     {
       Assert.That (
-          () => Assertion.IsNotNull<object> (null),
+          () => Assertion.IsNotNull<object?> (null),

```

Comment:
shouldn't we write these as null! with the NFO to better express the semantics of the test?